gtk4/src/expression.rs

```diff
@@ -146,6 +146,29 @@ impl Expression {
             ))
         }
     }
+
+    // rustdoc-stripper-ignore-next
+    /// Create a [`gtk::PropertyExpression`] that looks up for `property_name`
+    /// with self as parameter. This is useful in long chains of [`gtk::Expressions`].
+    pub fn chain_property(
+        &self,
+        this_type: glib::Type,
+        property_name: &str,
+    ) -> crate::PropertyExpression {
+        crate::PropertyExpression::new(this_type, Some(self), property_name)
+    }
```

Comment:
Can 't we make it work by passing doing something like this `chain_property<T>(property_name)` ? I think you can use the `IsA<Object>` for that :)